### PR TITLE
docs: Add troubleshooting notes for neo4j container

### PR DESCRIPTION
**Pull request by prafullkotecha**
### Tests

Additional test not needed, since the changes are done to documentation only.

### Documentation

- Add troubleshooting note for Neo4j docker container on Windows

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following instructions are for setting up a version of Amundsen using Docker
     If it's your first time, you may want to proactively go through [troubleshooting](troubleshooting) steps, especially the first one related to heap memory for ElasticSearch and Docker engine memory allocation (leading to Docker error 137).
 4. Ingest provided sample data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
 
-   * In a separate terminal window, change directory to [databuilder](https://github.com/amundsen-io/amundsen/tree/main/databuilder).
+   * In a separate terminal window, change directory to the [amundsendatabuilder](https://github.com/amundsen-io/amundsendatabuilder) submodule.
```
Comment (mgorsk1) [inline docs/installation.md:22]: I think this is old reference (should be to monorepo)

**Review by mgorsk1 — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 1 above)
Comment (prafullkotecha) [reply docs/installation.md:22]: I created this `patch-2` branch off of `main` branch. Should I use `master` branch instead?

Comment (dorianj) [reply docs/installation.md:22]: this was changed very recently by a different PR, i'd try rebasing of freshest main

Comment (prafullkotecha) [reply docs/installation.md]: Fixed. Sorry about the confusion and inconvenience.

**Review by dorianj — APPROVED** [review]

**Comment by dorianj** [comment]
@mgorsk1 can you approve/land please?

**Comment by prafullkotecha** [comment]
@dorianj - Could someone else review this?

**Comment by dorianj** [comment]
since @mgorsk1 requested changes, I unfortunately can't override it -- if you open as a new PR, we can land (sorry)

**Review by mgorsk1 — APPROVED** [review]
sorry, completely missed it!

**Comment by prafullkotecha** [comment]
@mgorsk1 @dorianj - Can you please approve the workflow. I just merged the upstream `main` branch onto this PR branch.

**Pull request merged**